sound/soc/sof/trace.c

```diff
@@ -236,6 +256,8 @@ int snd_sof_init_trace(struct snd_sof_dev *sdev)
 
 	init_waitqueue_head(&sdev->trace_sleep);
 
+	init_waitqueue_head(&sdev->trace_sleep);
```

Comment:
why are we initializing the waitqueue twice?

Reply:
sorry, this is refining error with 'patch am', will fix it, thanks.